src/pyrqlite/connections.py

```diff
@@ -40,11 +40,12 @@ class Connection(object):
         NotSupportedError,
     )
 
-    def __init__(self, host='localhost', port=4001,
+    def __init__(self, scheme='http', host='localhost', port=4001,
```

Comment:
With this change, perhaps you need the option to allow trusting of unsigned, or self-signed certs.

Reply:
And it would be interesting to have the unit tests generate some certs and use them.